src/main/kotlin/no/nav/familie/tilbake/faktaomfeilutbetaling/FaktaFeilutbetalingService.kt

```diff
@@ -74,7 +74,7 @@ class FaktaFeilutbetalingService(
             FaktaFeilutbetaling(
                 behandlingId = behandlingId,
                 perioder = feilutbetaltePerioder,
-                begrunnelse = hentAutomatiskSaksbehandlingBegrunnelse(behandling.saksbehandlingstype),
+                begrunnelse = behandling.begrunnelseForTilbakekreving
```

Comment:
Stemmer denne for Constants.AUTOMATISK_SAKSBEHANDLING_BEGUNNLESE ?

Reply:
Denne brukte vi tidligere direkte for de som kjøres automatisk. Vil nok fungere for oss, vi sender jo alltid inn denne, men usikker på hvordan dette vil fungere for BA/KS?